resource/ui/ts/models/proto.ts

```diff
@@ -113,6 +155,32 @@ module Models {
             available_range_count: number;
         }
 
+        /**
+         * Status is the common interface shared by NodeStatus and StoreStatus.
+         */
+        export interface Status{
```

Comment:
They can but it's not explicitly necessary, it will be asserted any time you try to assign a NodeStatus to something that expects a Status.